src/emulator/auth/apiSpec.js

```diff
@@ -2981,6 +2981,52 @@ export default {
   },
   components: {
     schemas: {
+      GoogleCloudIdentitytoolkitV1Argon2Parameters: {
```

Comment:
I see that #3732 is also updating apiSpec.js and schema.ts, and it seems like the changes are the same across the PRs. LMK if you would prefer I leave this out so it's easier to merge @yuchenshi

Reply:
Please leave those out

Reply:
Cool thanks, reverted!